config/Dockerfiles/linchpin/bin/run

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+id
+whoami
+exec tail -f /dev/null
```

Comment:
do we really want this container to stick around if they don't specify a command?

Reply:
This is based upon what ari et al. has done. I didn't have a better option. However, one of the things that could be done is to do a `docker exec` if the container is running in this way. Not that using `docker run -it ...` is a bad way to go. But more options seems better to me. Plus, the ability to run the container in openshift was that you'd likely run it with `docker exec` aiui from discussions with @arilivigni .

Reply:
ok - fair enough.